src/util.cpp

```diff
@@ -193,6 +193,17 @@ fs::path GetDebugLogPath()
     return AbsPathForConfigVal(logfile);
 }
 
+bool ShouldOpenDebugLog()
```

Comment:
Feels non-obvious based on the name that this would toggle the state of a global variable. Now that the bool option parsing is fixed, I don't think there's a need for this function. But even if you want to keep it, I think it would be clearer if the code in `AppInitMain` set `fPrintToDebugLog = ShouldOpenDebugLog()`.